CMakeLists.txt

```diff
@@ -99,8 +99,9 @@ target_compile_features(wildmeshing_toolkit PUBLIC cxx_std_17)
 
 #target_compile_options(wildmeshing_toolkit PUBLIC  -fconcepts)
 
+igl_include(restricted triangle)
 
-target_link_libraries(wildmeshing_toolkit PRIVATE wmtk::warnings)
+target_link_libraries(wildmeshing_toolkit PUBLIC wmtk::warnings igl_restricted::triangle)
```

Comment:
Do not change anything that you don't need to change. Keep wmtk::warnings private:
```
target_link_libraries(wildmeshing_toolkit PRIVATE wmtk::warnings)
target_link_libraries(wildmeshing_toolkit PUBLIC igl_restricted::triangle)
```

Additionally, I think that _triangle_ is only required in the components. So it would make sense to only add it to the components library.